src/sys/unix/selector/epoll.rs

```diff
@@ -212,6 +217,7 @@ pub mod event {
     }
 }
 
+#[cfg(target_os = "linux")]
```

Comment:
This needs to be `android`.

Reply:
Since it was on Android (for older versions) where `EPOLL_CLOEXEC` is missing, and only on Linux where this constant equality is assured, I thought it made more sense to perform the test on the platform where it was guaranteed to have the proper context.

Reply:
The point of the test was to ensure that `O_CLOEXEC` equals `EPOLL_CLOEXEC` on all platforms, as I couldn't find any documentation that those constants were guaranteed to be equal. But with this change we don't use `O_CLOEXEC` any more on all platforms except for Android, so we still need to check it on later versions of Android that this assertion is true.

Reply:
Fixed